orchagent/intfsorch.cpp

```diff
@@ -1042,6 +1076,10 @@ bool IntfsOrch::addRouterIntfs(sai_object_id_t vrf_id, Port &port)
     attr.value.u32 = port.m_mtu;
     attrs.push_back(attr);
 
+    attr.id = SAI_ROUTER_INTERFACE_ATTR_ADMIN_MPLS_STATE;
```

Comment:
Should we check if mpls is supported similar to NAT below?

Reply:
::addRouterIntfs has been updated to only include SAI_ROUTER_INTERFACE_ATTR_ADMIN_MPLS_STATE if non-default (ie, enable) is configured.  Explicit configuration to enable MPLS presumes platform that supports MPLS.  MPLS attr is no longer set on platforms unless it is explicitly part of the configuration.

Reply:
I'm puzzled that what the attribute SAI_ROUTER_INTERFACE_TYPE_MPLS_ROUTER is used for ? I didn't find it in current SWSS code. @qbdwlr